### DISCUSSION TITLE
Prowler vs Cloud Native Tools

### DISCUSSION THREAD

**Discussion by rajiv-bansal** [prowler-cloud/prowler#4025] [Q&A] [answered]
Hi,
I am a cloud architect and my role include advising clients for cloud security tools. I would like to understand what advantages does Prowler provides in comparison to cloud native tools. For AWS, I see that prowler is providing coverage for more regulatory benchmarks, but AWS is adding more benchmarks too. On Azure side, Azure is already covering a lot of benchmarks.

I want to understand the vision for using prowler - will it complement the native tools or be a better tool in comparison?

Regards

**Comment by toniblyx** [comment]
Thanks @rajiv-bansal, when you talk about cloud native tools which ones are you talking about?
- AWS Security Hub
- Azure Sentinel
- ?

**Comment by puchy22** [comment]
Hi @rajiv-bansal, it would be good if you could specify your question better so that it can be answered correctly, if you do not answer in a few weeks we will close the discussion so that you can reopen it when you need it. Thanks for everything

**Comment by puchy22** [comment, marked as answer]
Hi @rajiv-bansal,

We have recently added an answer to your question to the [documentation](https://docs.prowler.com/comparisons/), so this GitHub discussion will now be closed. If you have any further questions or need additional clarification, please feel free to reopen the discussion. Thank you!
